### ISSUE TITLE
CORS for staging & production S3

### ISSUE THREAD

**Issue by Gregoor** [mozilla/voice-web#1299]
```xml
<CORSConfiguration>
    <CORSRule>
        <AllowedOrigin>*</AllowedOrigin>
        <AllowedMethod>GET</AllowedMethod>
        <AllowedHeader>Authorization</AllowedHeader>
    </CORSRule>
</CORSConfiguration>
```

To normalize audio volumes we need to be able to handle the audio file in JS, hence we need CORS.

Unlocks PR #1285

**Comment by Gregoor** [comment]
@gozer any update on this? 🙃

**Comment by gozer** [comment]
Apologies, was blocked waiting on nubisproject/nubis-terraform#249

**Comment by gozer** [comment]
Fixed by 655887227554524ff1ede50ae35a4ef0da415d58
